src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/gadgets/Multimeter.java

```diff
@@ -43,17 +43,12 @@ public ItemUseHandler getItemHandler() {
             if (e.getClickedBlock().isPresent() && block.isPresent()) {
                 SlimefunItem item = block.get();
 
-                if (item instanceof EnergyNetComponent component && component.isChargeable()) {
-                    e.cancel();
+                if (item instanceof EnergyNetComponent component) {
+                    Location location = e.getClickedBlock().get().getLocation();
+                    Player player = e.getPlayer();
 
-                    Location l = e.getClickedBlock().get().getLocation();
-                    String stored = NumberUtils.getCompactDouble(component.getCharge(l)) + " J";
-                    String capacity = NumberUtils.getCompactDouble(component.getCapacity()) + " J";
-
-                    Player p = e.getPlayer();
-                    p.sendMessage("");
-                    Slimefun.getLocalization().sendMessage(p, "messages.multimeter", false, str -> str.replace("%stored%", stored).replace("%capacity%", capacity));
-                    p.sendMessage("");
+                    e.cancel();
```

Comment:
nit: I'd put this at the start/start of the if, bit weird to have this in the middle (and easy to get lost -- "is this cancelled?" - not quick to find when surrounded by code)